src/igvfd/tests/test_type_analysis_step_version.py

```diff
@@ -0,0 +1,21 @@
+import pytest
```

Comment:
Can the file be renamed? For example we have "[test_types_sample.py](https://github.com/IGVF-DACC/igvfd/blob/242c00497c1cd7b4c7ba978f715002041943c3a0/src/igvfd/tests/test_types_sample.py)" (types is plural not "type")